package.json

```diff
@@ -12,7 +12,8 @@
     "csp:configure": "cd libcsp && python3 waf configure --with-os=posix --enable-SDR --enable-rdp --enable-crc32 --enable-hmac --enable-xtea --with-loglevel=debug --enable-debug-timestamp --enable-python3-bindings --with-driver-usart=linux --enable-examples",
     "csp:build": "cd libcsp && python3 waf build",
     "csp:install": "cd libcsp &&  python3 waf install",
-    "build": "yarn csp:configure && yarn csp:build",
+    "uhf:build": "sudo rm -rf ex2_uhf_software && git clone https://github.com/AlbertaSat/ex2_uhf_software && cd ex2_uhf_software && git checkout ground_station_rebased_mods && ./build.sh",
```

Comment:
ground_station_rebased_mods has been merged. Shouldn't this just use master?

Reply:
Oh yeah, you're right... my bad

Reply:
Committed change